src/xrpld/core/JobTypes.h

```diff
@@ -53,17 +53,22 @@ class JobTypes
                        int limit,
                        std::chrono::milliseconds avgLatency,
                        std::chrono::milliseconds peakLatency) {
-            assert(m_map.find(jt) == m_map.end());
-
-            auto const [_, inserted] = m_map.emplace(
-                std::piecewise_construct,
-                std::forward_as_tuple(jt),
-                std::forward_as_tuple(
-                    jt, name, limit, avgLatency, peakLatency));
-
-            assert(inserted == true);
-            (void)_;
-            (void)inserted;
+            ASSERT(
+                m_map.find(jt) == m_map.end(),
+                "ripple::JobTypes::JobTypes::add : unique job type input");
+
+            [[maybe_unused]] auto const inserted =
+                m_map
+                    .emplace(
+                        std::piecewise_construct,
+                        std::forward_as_tuple(jt),
+                        std::forward_as_tuple(
+                            jt, name, limit, avgLatency, peakLatency))
+                    .second;
+
+            ASSERT(
```

Comment:
As before, would this be a candidate for using `SOMETIMES`, if you don't know if it's actually used?

Reply:
No, `SOMETIMES` is only a hint for the fuzzer, it does not check anything.